java/arcs/core/storage/Handle.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.storage
+
+import arcs.core.crdt.CrdtData
+import arcs.core.crdt.CrdtOperation
+import arcs.core.crdt.internal.VersionMap
+
+/**
+ * The [Callbacks] interface is a simple stand-in for the callbacks that a [Handle] might want
+ * to use* to signal information back to the runtime.
+ *
+ * This is mostly a place-holder at the moment.
+ */
+interface Callbacks {
+    fun onUpdate(handle: Handle<*, *, *>)
+}
+
+/**
+ * Base implementation of Arcs handles on the JVM runtime.
```

Comment:
Nit: this says "JVM runtime", but is part of the `arcs.core` package, which is platform-agnostic (or is supposed to be)

Reply:
Good point. There's still a bit of churn in my mind around what's destined for where. :-)

I'll just remove the `JVM` here, I think that specifying "runtime" (vs PEC) gets the point across enough.